joomla/joomla-cms
# [4.0] fix the path for Cassiopeia js

Pull Request for Issue # .

### Summary of Changes
- Moved the file `menu-metismenu.es6.js` from `build/media_source/templates/cassiopeia/js/mod_menu/` to `build/media_source/templates/site/cassiopeia/js/mod_menu/` (there is a `site` missing there)
- Adjust the layout to use the correct path

### Testing Instructions

Enable the dropdown layout for the frontend menu. Check that the metismenu script is loaded


### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required


@wilsonge this is kinda important for the Child templates
@richard67 I think you need to add the old paths to the files removed
